lib/repl.js

```diff
@@ -0,0 +1,64 @@
+var net = require("net");
+var repl = require("repl");
+var fs = require('fs');
+var path = require('path');
+var options = require('./args');
+var slice = require('slice-file');
+
+var ctrlC = new Buffer([ 255, 244, 255, 253, 6 ]).inspect();
+
+function bounce() {
+    var pidFile = path.join(options.tmp, 'registry-static.pid');
+    var self = this;
+    fs.readFile(pidFile, 'utf8', function(err, pid){
+        self.outputStream.write('Restarting process: '+pid+'\n');
+        process.kill(pid, 'SIGHUP');
+        self.displayPrompt();
+    });
+}
+
+var bounceDef = {
+    help: 'bounce the child process',
+    action: bounce
+};
+
+function liveLog() {
+    if (!options.log) {
+        this.outputStream.write("\nThe `log()` command only works when registry-static is invoked with a `--log` argument\n");
+        this.displayPrompt();
+    } else {
+        var tail = slice(options.log);
+        tail.follow(-10).pipe(this.outputStream);
+
+        // outputSteam on data?? yes. it's the socket.
+        this.outputStream.on('data', function dataListener(data){
+            // otherwise the repl gets into a weird state
+            if (ctrlC === data.inspect()) {
+                tail.close();
+            }
+        });
+    }
```

Comment:
Part of the problem here is that the repl server runs on the parent process, while there are logger instances in both the parent and child. The repl needs to be on the parent so that it can send the kill signal to the child and keep the repl open. Also, sometimes the logger logs to stdout, which is shared by both procs, or to a file handle, which is separate for each (but the same file), so 

The repl could be run on the child instead, but then the tradeoff is that when you `.restart`, you lose your telnet session. Not a huge deal, but the other thing you lose (if using this to tie into the child's logger) is anything from the parent logger (e.g. "Restarting..", etc.).

If that tradeoff is OK, then I'll go for that. Otherwise some pipe/stream fun will have to be had.